hypervisor/common/ptdev.c

```diff
@@ -105,22 +103,21 @@ alloc_entry(struct acrn_vm *vm, uint32_t intr_type)
 
 	INIT_LIST_HEAD(&entry->softirq_node);
 
-	initialize_timer(&entry->intr_delay_timer, ptdev_intr_delay_callback,
+	initialize_timer(&entry->intr_delay_timer, ptirq_intr_delay_callback,
 		entry, 0UL, 0, 0UL);
 
 	atomic_clear32(&entry->active, ACTIVE_FLAG);
 
 	return entry;
 }
 
-void
-release_entry(struct ptirq_remapping_info *entry)
+void ptirq_release_one_entry(struct ptirq_remapping_info *entry)
```

Comment:
Why you add "one" in release, but no "one" in alloc API?

Reply:
will use ptirq_release_entry